### ISSUE TITLE
Add examples for loading a jar after the JVM is started

### ISSUE THREAD

**Issue by Thrameos** [jpype-project/jpype#504]
A common issue that I see in forums is trying to add something to the classpath after the JVM has already be started.   As we can't restart the JVM that is already in progress, it can be an issue if multiple modules are using JPype.  Once a module has started the JVM, then nothing can add to the classpath leading to a fail.  I am not sure if there is a good procedure for this case in terms of getting the existing ClassLoader to change paths, but there is no reason that we can't create a second classloader for a runtime supplied path and use it for loading.

We should work through an example of this common problem and see how arduous it is.  If the pattern is very difficult then we should see if some additional support of needed in the library to simplify it. 

Not sure whether this is a long term or short term project.  If user respond to request this feature, I can move it up on the schedule.  Otherwise it will happen as part of the next documentation pass.

**Comment by pelson** [comment]
I think this would be a very worthwhile thing to document (and potentially provide an API/helper for). Interestingly there is a nice answer at https://stackoverflow.com/a/34111824/741316 which ironically uses JPype and IPython to interact with the classpath.

**Comment by Thrameos** [comment]
I have largely forgotten about this one.  I have worked a bit with class loaders since this original perhaps I should cycle back to this.

**Comment by pelson** [comment]
There was an addition of ``addClassPath`` in v1.2.0 which helps with this. It isn't really documented properly though (it is only mentioned in a table in https://jpype.readthedocs.io/en/latest/quickguide.html#starting-jpype)

**Comment by Thrameos** [comment]
Yes addClassPath allows adding of classes after the jvm is started.  But to has weird restrictions.   Certain classes such as sql drivers can only be loaded by the application loader.  Basically the security model only recognizes classes loaded by the same classs loader.   I wanted to make it so all classes used the dynamic class loader, but then sql does not work.   So we end up with two tiers of classes,  the priviliged loaded with jvm and the second tier loaded after.  Mixing the two gave odd results hence the difficulty I had with documenting it.  For non sql it is best if all went with the dynamic so less mixing.  But hard to achive that without starting jvm then calling addclasspath to deliberately force them as dynamic.

**Comment by pelson** [comment]
I got curious how it would look to have a dynamic class loader, and put together a simple self contained example https://gist.github.com/pelson/0bea07edb6f4693de3d38bd4c3d53a82


In short, this seems to work very nicely.
